test/test_no_regression.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+from walter import project
+import pandas,os
+from walter.data_access import get_data_dir
+from pathlib2 import Path
+import numpy.testing as np
+
+def test_same_result_zerolight():
```

Comment:
I do not understand why there is "zerolight" in the name of the test ? Maybe the test could be called test_same_result() to make its purpose clearer ?